pkg/contract/solana/gateway.go

```diff
@@ -1,5 +1,9 @@
 package solana
```

Comment:
The parent package should be named `contracts`, not `contract`

Reply:
https://github.com/zeta-chain/node/pull/2560/commits/e94fd92e535981bea13dd89a2ad4e13a90a9c147